settings/BUILD.bazel

```diff
@@ -1,10 +1,11 @@
 load("@bazel_skylib//rules:common_settings.bzl", "bool_setting", "string_setting")
 
+package(default_visibility = ["//:__subpackages__"])
```

Comment:
Thanks. I had discovered the flag was enabled in the BCR in CI but didn't get the overlay changes pushed back to this repo.